test/HubPool.Fixture.ts

```diff
@@ -1,7 +1,7 @@
 import { TokenRolesEnum } from "@uma/common";
 import { getContractFactory, randomAddress, toBN, fromWei } from "./utils";
 
-import { bondAmount, refundProposalLiveness, finalFee, identifier, repaymentChainId } from "./constants";
```

Comment:
we should lint for unused variables.

Reply:
Agree! We should turn on eslint, as it will catch stuff like this. I think these are all used, though.